dlpy/utils.py

```diff
@@ -2545,6 +2578,52 @@ def create_object_detection_table_no_xml(conn, data_path, coord_type, output, an
                 run;
                 '''.format(output, string_input_tbl_name)
     conn.runcode(code=fmt_code, _messagelevel='error')
+    
+    # Verify that bounding boxes meet user requirements. (Only YOLO coordinate_type currently supported)
+    if check_bbox == True:
+        if coord_type.lower() == 'yolo':
```

Comment:
YOLO format takes [xMiddle, yMiddle, width, height] in normalized image size, while YOLO format takes [xMin, yMin, xMax, yMax]. Since all labels will be converted to YOLO format for Yolo v2 algorithms, it might be better to use [min_bbox_width, min_bbox_height] as criteria, as these two parameters are invariant of image dimensions.

Reply:
Again, I don't really understand your comment.
For clarification:
COCO format = xMin, yMin, xMax, yMax
YOLO format = xMiddle, yMiddle, width, height

Your propose to use yolo format for user criteria?
E.g. I should define minimum width = 0.02?

Then my answer is:
I chose to convert all coordinates into COCO format (after resizing of the image) and drop them based on COCO values for the following reasons:
1. Training errors occur if your object is too small. A value of width=0.02 for 832x832 images might work while it won't work for 416x416 images. I think the minimum width value for training is dependent on the image size.
2. If you have images of different sizes, it will be impossible for a human to estimate how small/large the objects will be in his training dataset.

Reply:
Bounding boxes are only evaluated and dropped if coord_type.lower() == 'yolo', but the parameters you're using are those of COCO format. Could you verify it?